Build issues / undocumented use with catkin_virtualenv

Hello @ablakey 

I was inspired by your ROSCon talk to recreate the wifi-heatmap with one of my robots but have run into issues while building the package. 
I have installed qgis as listed in the instructions, but I cannot build the qgis_ros package.
I cloned and build catkin_virtualenv since it is listed as a dependency for qgis_ros (but this is not stated on in the readme file), but I am unsure if I need to do anything else in order to build the package.

I get the following error messages when building. Hope you can help!

```
Starting  >>> catkin_virtualenv                                                
Finished  <<< catkin_virtualenv                [ 0.1 seconds ]                 
Starting  >>> qgis_ros                                                         
_______________________________________________________________________________
Errors     << qgis_ros:make /home/kajep09/RCW_workspaces/safecop_ws/logs/qgis_ros/build.make.003.log
Traceback (most recent call last):
  File "/home/kajep09/RCW_workspaces/safecop_ws/src/mobile/utility_nodes/catkin_virtualenv/catkin_virtualenv/cmake/build_venv.py", line 80, in <module>
    deploy.create_virtualenv()
  File "/home/kajep09/RCW_workspaces/safecop_ws/src/mobile/utility_nodes/catkin_virtualenv/catkin_virtualenv/src/dh_virtualenv/deployment.py", line 169, in create_virtualenv
    Traceback (most recent call last):
subprocess.check_call(virtualenv)
  File "/usr/lib/python2.7/subprocess.py", line 536, in check_call
  File "/home/kajep09/RCW_workspaces/safecop_ws/src/mobile/utility_nodes/catkin_virtualenv/catkin_virtualenv/cmake/build_venv.py", line 80, in <module>
    deploy.create_virtualenv()
  File "/home/kajep09/RCW_workspaces/safecop_ws/src/mobile/utility_nodes/catkin_virtualenv/catkin_virtualenv/src/dh_virtualenv/deployment.py", line 169, in create_virtualenv
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 523, in call
    subprocess.check_call(virtualenv)
  File "/usr/lib/python2.7/subprocess.py", line 536, in check_call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 523, in call
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
    raise child_exception
OSError: [Errno 2] No such file or directory
make[2]: *** [/home/kajep09/RCW_workspaces/safecop_ws/devel/.private/qgis_ros/share/qgis_ros/venv] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [venv] Error 1
make[1]: *** [CMakeFiles/qgis_ros_generate_virtualenv.dir/all] Error 2
make: *** [all] Error 2
cd /home/kajep09/RCW_workspaces/safecop_ws/build/qgis_ros; catkin build --get-env qgis_ros | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd -
...............................................................................
Failed     << qgis_ros:make                    [ Exited with code 2 ]          
Failed    <<< qgis_ros                         [ 0.1 seconds ]                 
[build] Summary: 1 of 2 packages succeeded.                                    
[build]   Ignored:   76 packages were skipped or are blacklisted.              
[build]   Warnings:  None.                                                     
[build]   Abandoned: None.                                                     
[build]   Failed:    1 packages failed.                                        
[build] Runtime: 0.4 seconds total.                                            
* Sourced workspace devel/setup.bash
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,15 +2,9 @@ cmake_minimum_required(VERSION 2.8.3)
 project(qgis_ros)
 
 find_package(catkin REQUIRED COMPONENTS
-  catkin_virtualenv
 )
 
 catkin_package()
-
-catkin_generate_virtualenv(
-    PYTHON_VERSION_MAJOR 3
-    USE_SYSTEM_PACKAGES FALSE
-)
 
 catkin_python_setup()
 

--- a/package.xml
+++ b/package.xml
@@ -9,12 +9,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>catkin_virtualenv</build_depend>
   <depend>nav_msgs</depend>
   <depend>geometry_msgs</depend>
-
-  <export>
-    <pip_requirements>requirements.txt</pip_requirements>
-  </export>
 
 </package>

